src/ray/gcs/client.cc

```diff
@@ -2,10 +2,80 @@
 
 #include "ray/gcs/redis_context.h"
 
+/* Parsing the return string with Regex to get IP address and port */
```

Comment:
We are using `/* */` in the legacy Ray codebase, but this should be converted to `//`.